### PR TITLE
Removed saveQuietly

### PR DESCRIPTION
**Pull request by yasamprom**
Closes objectionary/eo#1248

### CODE REVIEW

**Comment by yasamprom** [comment]
@mximp I have a question about `SpyTrain.java` implementation. I throw raw exception and it is bad practice. As I understood there is no way to throw checked exception in lambda. Is my implementation ok or you may suggest better way?

File: eo-maven-plugin/src/main/java/org/eolang/maven/SpyTrain.java [hunk 1]
```diff
@@ -55,12 +57,16 @@ final class SpyTrain extends TrEnvelope {
...
-                            );
+                            try {
+                                new Home().save(
+                                    xml.toString(),
+                                    dir.resolve(
+                                        String.format("%02d-%s.xml", pos, log)
+                                    )
+                                );
+                            } catch (final IOException exc) {
+                                throw new RuntimeException(exc);
```
Comment (mximp) [inline eo-maven-plugin/src/main/java/org/eolang/maven/SpyTrain.java:68]: @yasamprom Maybe we can use here `IllegalStateException` as it was in method `saveQuietly`?

Comment (yasamprom) [reply eo-maven-plugin/src/main/java/org/eolang/maven/SpyTrain.java]: Hm, yes. Thanks!

**Comment by yasamprom** [comment]
@yegor256 Please see

**Comment by yegor256** [comment]
@yasamprom let's do this instead: https://github.com/yegor256/xsline/issues/40 It will help us and all other projects that use Xsline

**Review by mximp — APPROVED** [review]

**Comment by yasamprom** [comment]
@yegor256 Let's merge this PR as its code works correctly and assign yegor256/xsline#40 to someone to implement this in future

**Comment by yegor256** [comment]
@yasamprom sure, we can do this, but you have to add a puzzle to the code, so that we don't forget

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**